### PR TITLE
Update README file

### PR DESCRIPTION
**Pull request by Tomorrow9**
Signed-off-by: dianew <dianew@vmware.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,14 +32,15 @@
 * ansible.cfg: User customized Ansible configuration file
 * autoinstall: Folder for guest OS unattend install configuration files
 * common: Folder for common tasks called in test cases
-* docs: Folder for guide files
+* docs: Folder for guide file and known issues
 * env_setup: Folder for playbooks or tasks which to prepare or cleanup testing environment
 * linux: Folder for playbooks to test Linux guest OS
 * Windows: Folder for playbooks to test Windows guest OS
```
Comment (keirazhang) [inline README.md:38]: The first letter of windows folder should be lower case

File: README.md [hunk 2]
```diff
@@ -32,14 +32,15 @@
...
 * plugin: Folder for plugin scripts
 * tools: Folder for 3rd-party tools used in test cases
 * vars: Folder for variable files used in testing
 * logs: Folder generated by default for files collected and logs generated in test run
```
Comment (keirazhang) [inline README.md:42]: There is no logs folder from source code. Maybe we can add a step to tell user where to find logs in Steps to Launch Testing after running playbook.

File: README.md (hunk 1 above)
Comment (Tomorrow9) [reply README.md]: Right, modified.

File: README.md (hunk 2 above)
Comment (Tomorrow9) [reply README.md]: Removed this "logs" line, and added to steps 5 above. Thanks.

**Review by keirazhang — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**